src/ontology/mondo.Makefile

```diff
@@ -681,6 +681,15 @@ test_owlaxioms:
 
 test: test_owlaxioms 
 
+.PHONY: test_obs_reason
+test_obs_reason:
+	echo "all obsolesence reasons should be typed as xsd:string"
+	! grep -E "property_value: IAO:0000231.*xsd:[^s]" mondo-edit.obo
+	echo "obsolesence reason does not seem to use the correct property"
+	! grep -E "\"terms merged\" xsd:" mondo-edit.obo | grep -v IAO:0000231
```

Comment:
```suggestion
	! grep -E "\"terms merged\" xsd:" mondo-edit.obo | grep -v IAO:0000231
	! grep -E "\"out of scope\" xsd:" mondo-edit.obo | grep -v IAO:0000231
	! grep -E "\"termssplit\" xsd:" mondo-edit.obo | grep -v IAO:0000231
```

Reply:
termssplit one word? if so its fine.

Reply:
ha, no, that was a typo, thanks for catching that! I updated it